action/README.md

```diff
@@ -0,0 +1,33 @@
+# Functions framework conformance test javascript action
```

Comment:
Github Actions currently only supports writing actions in Javascript or in Docker containers -- we can't use the latter, since the conformance test will eventually run a Docker container as part of its run, and Github Actions Docker containers don't allow running in privileged mode (and therefore cannot run docker-within-docker).

Anyway, long story short, right now Javascript is the only supported language.